.github/workflows/ci-cd.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       s3mock:
         image: ghcr.io/project-zot/localstack/localstack:0.13.2
         env:
-          SERVICES: s3
+          SERVICES: s3,dynamodb
```

Comment:
This makes us a user of another aws service? :\

Reply:
yes..I propose to keep backing up boltdb in s3, and locally we just download it to a temp dir, use it, and keep backing it up?